pkg/ansible/proxy/cache_response.go

```diff
@@ -141,12 +141,40 @@ func (c *cacheResponseHandler) ServeHTTP(w http.ResponseWriter, req *http.Reques
 }
 
 // skipCacheLookup - determine if we should skip the cache lookup
-func (c *cacheResponseHandler) skipCacheLookup(r *requestfactory.RequestInfo) bool {
+func (c *cacheResponseHandler) skipCacheLookup(r *requestfactory.RequestInfo, gvk schema.GroupVersionKind, req *http.Request) bool {
 	// check if resource is present on request
 	if !r.IsResourceRequest {
 		return true
 	}
 
+	owner, err := getRequestOwnerRef(req)
+	if err != nil {
+		log.Error(err, "Could not get owner reference from proxy.")
+		return false
+	}
+	ownerGV, err := schema.ParseGroupVersion(owner.APIVersion)
+	if err != nil {
+		m := fmt.Sprintf("Could not get group version for: %v.", owner)
+		log.Error(err, m)
+		return false
+	}
+	ownerGVK := schema.GroupVersionKind{
+		Group:   ownerGV.Group,
+		Version: ownerGV.Version,
+		Kind:    owner.Kind,
+	}
+
+	// Make sure that the gvk is not blacklisted
+	relatedController, ok := c.cMap.Get(ownerGVK)
+	if !ok {
+		log.Info("Could not get related controller from cmap. ", "Owner GVK:", ownerGVK)
```

Comment:
Users won't know what `cmap` is, maybe something like, `could not find watch for gvk` would make more sense to the user